src/sdk/namespace_fs.js

```diff
@@ -914,13 +978,14 @@ class NamespaceFS {
 
     async upload_multipart(params, object_sdk) {
         try {
+            const open_mode = 'w';
```

Comment:
I would put a comment here that explains that we should switch to using `wt` here as well,
but it's definitely less of a priority because multipart uploads are large and the overhead of renaming a large part file is amortized in the entire upload of the part. However it still has a benefit in reducing rename contentions so we would like to close this todo at some point.

Reply:
done